src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -185,8 +185,9 @@ class SidebarLinks extends React.Component {
      * @param {String} option.reportID
      */
     showReportPage(option) {
-        if (this.props.isCreateMenuOpen) {
+        if (this.props.isCreateMenuOpen || this.props.currentReportID === option.reportID) {
             // Prevent opening Report page when click LHN row quickly after clicking FAB icon
+            // or when click the same LHN row
```

Comment:
Also, let's take this comment above the if block
```suggestion
            // or when clicking the active LHN row
```

Reply:
> Also, let's take this comment above the if block
@tienifr can we please also do this ^